client/my-sites/site-settings/subscriptions/index.jsx

```diff
@@ -77,6 +77,10 @@ const Subscriptions = ( {
 			<CompactCard href={ '/people/email-followers/' + selectedSiteSlug }>
 				{ translate( 'View your Email Followers' ) }
 			</CompactCard>
+
+			<CompactCard href={ 'https://wordpress.com/manage/' + selectedSiteId }>
+				{ translate( 'Migrate followers from another site' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Migrate followers from another WordPress.com blog' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).